docs/index.rst

```diff
@@ -2531,7 +2560,47 @@ you might need to import the following constants:
     from bigml.api import UNKNOWN
     from bigml.api import RUNNABLE
 
-You can query the status of any resource with the ``status`` method:
+Usually, you will simply need to wait until the resource is
+in the ``bigml.api.FINISHED`` state for further processing. If that's the case,
+the easiest way is calling the ``api.ok`` method and passing as first argument
+the object that contains your resource:
+
+.. code-block:: python
+
+    from bigml.api import BigML
+    api = BigML() # creates a connection to BigML's API
+    source = api.create_source('my_file.csv') # creates a source object
+    api.ok(source) # checks that the source is finished and updates ``source``
+
+In this code, the ``api.create_source`` will probably return a non-finished
```

Comment:
"the api.create_source will" sounds strange... i'd say either "api.create_source will" (sans "the"), or "the api.create_source call".